FileTemplates/YOView.xctemplate/___FILEBASENAME___.m

```diff
@@ -14,8 +14,14 @@ @interface ___FILEBASENAMEASIDENTIFIER___ ()
 @implementation ___FILEBASENAMEASIDENTIFIER___
 
 - (void)viewInit {
+
+    // Initialize subviews here
+
     __weak typeof(self) weakSelf = self;
```

Comment:
Yeah totally.

I kinda go back and forth on `YOSelf` vs `__weak typeof(self)`. Here's my thinking: I prefer to keep a consistent pattern across the codebase for weak references, and I'm not going to import `YOLayout.h` just for `YOSelf`. I've created a snippet that autocompletes on `__weak` for the weak reference pattern, so it doesn't really take any extra time.